manual/setup-geth-lodestar.md

```diff
@@ -0,0 +1,95 @@
+Instructions for manually setting up a node on Ephemery using geth and lodestar.
+
+### Prerequisites
+
+[Install golang](https://go.dev/doc/install) for your system.
+[Install Yarn](https://classic.yarnpkg.com/lang/en/docs/install/).
+Check that your [Node](https://nodejs.org/) version is >= v18.15.0.
+
+Then:
+```
+sudo apt install -y git default-jre make gcc
+```
+
+### Download config
+
+```
+mkdir testnet-all
+cd testnet-all
+```
+
+Download the `testnet-all.tar.gz` file for the [latest release of the Ephemery testnet](https://github.com/ephemery-testnet/ephemery-genesis/releases) to this directory using `wget`. Then unzip inside this folder.
+```
+tar -xzf testnet-all.tar.gz
+```
+
+### Generate jwt
+
+```
+openssl rand -hex 32 | tr -d "\n" > "/tmp/jwtsecret"
+```
+
+### Execution Layer
+
+Download and build software:
+```
+cd ~
+git clone https://github.com/ethereum/go-ethereum.git
+cd go-ethereum
+make geth
+```
+Initialise:
+```
+cd ~
+./go-ethereum/build/bin/geth init --datadir "datadir-geth" ~/testnet-all/genesis.json
+```
+Run:
+```
+./go-ethereum/build/bin/geth \
+     --networkid {networkID} \
+     --syncmode=full \
+     --port 30303 \
+     --http \
+     --datadir "datadir-geth" \
+     --authrpc.jwtsecret=/tmp/jwtsecret \
+     --bootnodes {bootnodes}
+```
+
+For `{bootnodes}` look in ~/testnet-all/boot_enode.txt. Entries must be separated,by,commas and "enclosed in quotes".
```

Comment:
User can just source the file with variables, load it to the shell session and run the same command 

```
source node_vars.txt 
...
--bootneds $BOOTNODE_ENR_LIST
```

Reply:
Same - will update both